setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name = 'OpenFisca-Core',
-    version = '4.1.5',
+    version = '4.1.6',
```

Comment:
Adding a new public API method is not a patch bump, but a minor bump. This method did not exist beforehand, and is thus backward-incompatible.
See [SemVer](http://semver.org).